control_panel_api/services.py

```diff
@@ -175,6 +175,12 @@ def create_bucket(bucket_name):
         target_prefix=f"{bucket_name}/")
     aws.put_bucket_encryption(bucket_name)
 
+    if is_data_warehouse:
+        aws.put_bucket_tagging(
+            bucket_name,
+            tags={'buckettype': 'datawarehouse'}
```

Comment:
The OCD in me thinks `bucket-type`: `data-warehouse` :) - I need to double check, but it should be possible to use `-` here right?